README.md

```diff
@@ -402,14 +402,69 @@ To learn by examples, see `examples/scripts` to have a feeling of how to script
 
 ### Logging
 
-We used to have a node-wot-logger package to allow fine-grained logging (by means of Winston). It turned out though that depending on the actual use-case other logging libraries might be better suited. Hence we do not want to prescribe which logging library to use. Having said that, we use console statements which can be easily overriden to use the prefered logging library if needed (see [here](https://gist.github.com/spmason/1670196)).
+Logging in node-wot is implemented via the [`debug`](https://www.npmjs.com/package/debug) package.
+This allows users to enable log messages for specific logging levels (`info`, `debug`, `warn`, or `error`) or packages.
+Which log messages are emitted is controlled by the `DEBUG` environment variable.
 
-The logs in the library follows those best practice rules (see [here](https://github.com/eclipse/thingweb.node-wot/issues/229)):
+In the following, we will show a couple of examples for its usage using wildcard characters (`*`).
+Note, however, that the syntax for setting an environment variable depends on your operating system and the terminal you use.
+See the [`debug` documentation](https://www.npmjs.com/package/debug) for more details on platform-specific differences.
 
-1. Tag log messages with the package as following: `console.debug("[package-name]", "log message)`. This is useful to identify which package generated the log.
-2. Avoid to use `info` and `log` in packages other than the cli package.
+#### Examples
 
-Please follows these rules if you are going to contribute to node-wot library.
+First, you can enable all log messages by setting `DEBUG` to a wildcard like so:
+
+```sh
+DEBUG=* npm start
+```
+
+To only show `node-wot`-specific logging messages, prefix the wildcard with `node-wot`:
+
+```sh
+DEBUG=node-wot* npm start
+```
+
+To only show a specific log level, use one of `info`, `debug`, `warn`, or `error` as the suffix.
+Note in this context that you can provide multiple values for `DEBUG`.
+For example, if you want to show only `debug` and `info` messages, you can use the following:
+
+```sh
+DEBUG='*debug,*info' npm start
+```
+
+Finally, you can choose to only display log messages from a specific `node-wot` package.
+For example, if you only want to see log messages for the `core` package, use the following:
+
+```sh
+DEBUG=node-wot:core* npm start
+```
+
+Using the log levels above, you can also apply more fine-grained parameters for logging.
+For instance, if you only want to see `error` messages from the `binding-coap` package, use this:
+
+```sh
+DEBUG=node-wot:binding-coap*debug npm start
```

Comment:
I think it should be 

DEBUG=node-wot:binding-coap*error npm start

Reply:
Thank you, fixed in 73a8da2f206ca232323ff5ca709ffe622ab358a0 :)